src/classes/EscalationTrigger.cls

```diff
@@ -193,6 +194,24 @@ public with sharing class EscalationTrigger extends BaseTrigger {
 	}
 
 	/**
+	* Assign default owner to an escalation
+	*/
+	@TestVisible
+	private void assignDefaultOwner() {
+		if (!this.isInsert) {
+			return;
+		}
+
+		rh_escal__EscalationRouter__c er = rh_escal__EscalationRouter__c.getValues(DEFAULT_ESCAL_ROUTER);
```

Comment:
This is not how we want to do this.  We want to select one based on the record type name and if that doesn't exist, then check for the default.

```java
public class RecordTypeUtils {
    public class UnknownRecordType extends Exception {}

    @TestVisible private static String MSG_UNKNOWN_RECORDTYPE = 'Unknown record type';

    @TestVisible static List<String> OBJECT_NAMES = new List<String> {
        'rh_escal__Escalation__c'
    };

    @TestVisible private static Map<Id, RecordType> RECORD_TYPE_MAP {
        get {
            if (RECORD_TYPE_MAP == null) {
                RECORD_TYPE_MAP = new Map<Id, RecordType>([
                    select Name
                    from RecordType
                    where SobjectType in :OBJECT_NAMES
                ]);
            }

            return RECORD_TYPE_MAP;
        }
        private set;
    }

    public static String getRecordTypeName(Id recordTypeId) {
        if (recordTypeId == null || !RECORD_TYPE_MAP.containsKey(recordTypeId)) {
            throw new UnknownRecordTypeException(MSG_UNKNOWN_RECORDTYPE);
        }

        return RECORD_TYPE_MAP.get(recordTypeId).Name;
    }
}
```
_Don't forget to add javadoc_

Then create a router utils class

```java
public class EscalationRouterUtils {
    @TestVisible private static String DEFAULT_ROUTER = 'DEFAULT';
    @TestVisible private static String MSG_NO_VALID_ROUTER = 'No valid router found for record type';

    public class UnknownRouterException extends Exception {}

    public static rh_escal__EscalationRouter__c getRouterForRecordType(Id recordTypeId) {
        rh_escal__EscalationRouter__c router;
        String recordTypeName;

        try {
            recordTypeName = RecordTypeUtils.getRecordTypeName(recordTypeId);
        } catch (RecordTypeUtils.UnknownRecordTypeException e) {
            System.debug(System.LoggingLevel.DEBUG, 'Could not find record type by id');
        }

        if (recordTypeName != null) {
            router = rh_escal__EscalationRouter__c.getValues(recordTypeName);
        }

        if (router == null) {
            router = rh_escal__EscalationRouter__c.getValues(DEFAULT_ROUTER);
        }

        if (router == null) {
            throw new UnknownRouterException(MSG_NO_VALID_ROUTER);
        }

        return router;
    }
}
```

Finally do this in your trigger class

```java
@TestVisible
private void assignDefaultOwner() {
    if (!this.isInsert) {
        return;
    }

    for (rh_escal__Escalation__c escalation : this.newObjs) {
        try {
            rh_escal__EscalationRouter__c router = EscalationRouterUtils.getRouterForRecordType(escalation.RecordTypeId);
            if (router.rh_escal__DefaultOwner__c != null) {
                escalation.OwnerId = router.rh_escal__DefaultOwner__c;
            }
        } catch (EscalationRouterUtils.UnknownRouterException e) {
            System.debug(System.LoggingLevel.DEBUG, 'Unable to find router for record type');
            continue;
        }
    }
}
```

I know this is a bit more convoluted, but it will mean that the logic is all contained and it will be usable for all installations.  I'm hopeful that we can just create the record type inside the test.  If we cannot then we'll have to either fudge the tests or just leave them out entirely.